vms/platformvm/txs/executor/camino_tx_executor_test.go

```diff
@@ -4663,18 +4677,27 @@ func TestCaminoStandardTxExecutorExportTxMultisig(t *testing.T) {
 	type test struct {
 		destinationChainID ids.ID
 		to                 ids.ShortID
+		isMSig             bool
 		expectedErr        error
 	}
 
 	tests := map[string]test{
 		"P->C export from msig wallet": {
 			destinationChainID: cChainID,
 			to:                 fakeMSigAlias,
+			isMSig:             true,
```

Comment:
suggest to replace it with expected shm utxos and make direct comparison with require - that will be 100% bulletproof